### DISCUSSION TITLE
Is there a way to add click count to each post pages of the blog?

### DISCUSSION THREAD

**Discussion by DayuanJiang** [quarto-dev/quarto-cli#5868] [Q&A]
### Description

I am currently utilizing Quarto to develop my blog and would like to display a list of post ranked by click count. Is there any method to accomplish this?

**Comment by mcanouil** [comment]
This is more of an HTML question than Quarto.
You'll have better luck using your favourite online search engine to find how to compute such statistics, such as Google Analytics (see <https://quarto.org/docs/websites/website-tools.html#google-analytics>).
